packages/server/lib/plugins/child/run_plugins.js

```diff
@@ -155,9 +155,16 @@ const runPlugins = (ipc, pluginsFile, projectRoot) => {
   })
 
   process.on('unhandledRejection', (event) => {
-    const err = (event && event.reason) || event
+    let err = event
+
+    debug('unhandled rejection:', event)
+
+    // Rejected Bluebird promises will return a reason object.
+    // OpenSSL error returns a reason as user-friendly string.
+    if (event.reason && typeof event.reason === 'object') {
```

Comment:
Want to check if the event is non null/undefined before checking reason? Dunno if it matters but it was checked before.
```suggestion
    if (event && event.reason && typeof event.reason === 'object') {
```